mypy/fastparse.py

```diff
@@ -186,7 +186,7 @@ def parse_type_ignore_tag(tag: Optional[str]) -> List[str]:
     # TODO: Implement proper parsing and error checking
     if not tag:
         return []
-    m = re.match(r'\[([^#]*)\]', tag)
+    m = re.match(r'\s*\[([^#]*)\]', tag)
```

Comment:
Ideally we should give a warning if user makes a mistake like `# type: ignore [one-code][other-code]` or something else we don recognize, but I see there is already a TODO about this.

Reply:
Yes, I'll add better error checking in a separate PR.